src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -388,6 +390,12 @@ private void validateOnlineCourseAndRegistrationEnabled(Course course) {
         }
     }
 
+    private void validateAccuracyOfScores(Course course) {
+        if (course.getAccuracyOfScores() == null) {
```

Comment:
I think we should also check that the value makes sense and exclude negativ values and values larger than 5 or 10